packages/editor/src/lib/editor/Editor.ts

```diff
@@ -9571,6 +9571,7 @@ export class Editor extends EventEmitter<TLEventMap> {
 				if (info.key === 'ShiftRight') info.key = 'ShiftLeft'
 				if (info.key === 'AltRight') info.key = 'AltLeft'
 				if (info.code === 'ControlRight') info.code = 'ControlLeft'
+				if (info.code === 'MetaRight') info.code = 'MetaLeft'
```

Comment:
Are we distinguishing between control and meta? We mix the two into the `info.ctrlKrey`. Is it possible to do this type of selection on Windows then?

Reply:
- meta/ctrl key is currently combined in our logic but i think it's...fine, generally speaking. so on windows, this technically might not be correct if you're holding down the Win key (meta) but it will work as expected for the Ctrl key on Windows
- so yeah, you can do this on Windows with Ctrl key.
- and double yes, Miro lets Cmd/Ctrl be additive

Reply:
ok, I'd like to try on a PC before landing just to be sure, but I think this will be good to land once we have tests up. Selection is fuuucking complicated, appreciate the patience while I go through the cases